release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3544,4 +3544,39 @@ For more information, see xref:../nodes/clusters/nodes-cluster-enabling-features
 [id="ocp-4-12-13-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-14"]
+=== RHBA-2023:1858 - {product-title} 4.12.14 bug fixes and features
```

Comment:
```suggestion
=== RHBA-2023:1858 - {product-title} 4.12.14 bug fixes 
```

Reply:
I was keeping things consistent with above and I'm not sure we've ever changed the titles to reflect features.

Reply:
Change made, thanks!